### ISSUE TITLE
2021-11-4

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#645]
261Logits(Response)-based Knowledge; Distilling the Knowledge in a Neural Network Hilton NIPS 2014; KD的开山之作，核心思想是使用softed labels去学习class distribution，具体先训练好一个teacher网络，然后将teacher的网的输出结果q作为student网络的目标，训练student网络，使得student网络的结果p接近q。提出的新idea是softmax的变形，引入一个变量T去产生softed labels。soft target 与hard target区别如下图所示。
262Deep mutual learning CVPR 2018 ; 传统蒸馏模型是从功能强大的大型网络或集成网络转移到结构简单，运行快速的小型网络。本文打破这种预先定义好的“强弱关系”，提出DML，即让一组学生网络在训练过程中相互学习、相互指导，而不是静态的预先定义好教师和学生之间的单向转换通路 ; 作者引入KL散度的概念来度量两个学生网络的输出概率p1和p2。相信了解过GAN网络的小伙伴对KL应该不会陌生，KL 散度是一种衡量两个概率分布的匹配程度的指标，两个分布差异越大，KL散度越大。作者采用KL散度，衡量这两个网络的预测p1和p2是否匹配 ; 
263博士研究生考试分为春季考试和秋季考试两种，一般来说春季考试在三月份进行，九月份入学；秋季考试在十或十一月份进行，次年三月份入学。 不过博士研究生考试是由各学校组织的，具体考试时间不同的学校也是不一样的。 有的学校是两次考试两次入学，有的是一次考试一次入学，还有的是一次考试两次入学。(2021-10-22)
264华中科技大学2021年博士研究生入学考试于2021年3月27日（本周六）下午14:00-17:00进行，本校考点为东九楼D栋五楼。因学校实行封闭管理，请来参加考试的考生凭“准考证+健康码”从东九门入校。因车位紧张，请各考生选择步行入校，进校后在东九楼D栋西门排队等待，并在现场工作人员引导下有序进入大楼。因需要排队入场并查验证件，请各考生尽量提前一小时到达考试现场。(2021-10-22)
265申请人在上海交通大学研究生招生网上报名，系统开放时间为2021年10月15日至12月15日或2022年3月15日至4月20日，具体报名截止时间请查阅学院的通知，根据学院要求准备申请材料（务请考生注意学院规定的申请时间）。 https://yzb.sjtu.edu.cn/zsjz/bsszs/bsszsjz.htm  (2021-10-22)
266Search HUST professional doctor; 
267一个典型的KD框架如下图所示，由三个part组成，Teacher model，student model 和Knowledge transfer，整个过程是在有监督的data数据集上训练完成。
268KD主要方法; 作者参考[3]这篇文章，从Logits(Response)-based, Feature-based, 和Relation-based knowledge三种维度去介绍KD在过去6年的一些高引用paper。 Logits(Response)-based knowledge从下图直观感受到，knowledge从teacher model的output layer学习得到；Feature-based 是从一些中间hidden layers学习knowledge；Relation-based则是学习input-hidden-output之间的关系。
269Logits(Response)-based Knowledge;
